properties/P000149.md

```diff
@@ -0,0 +1,13 @@
+---
+uid: P000149
+name: $\varepsilon$-space
+aliases:
+  - $\omega$-Lindelöf
+refs:
+- doi: 10.1016/0166-8641(82)90065-7
+  name: Some properties of C(X), I (Gerlits & Nagy)
+---
+A family $\mathcal U$ of open subsets of a topological space $X$ is called an $\omega$-cover if every finite subset of $X$ is contained in one of the members and $X \not\in \mathcal U$.
+
+A space $X$ is an $\varepsilon$-space if every open $\omega$-cover has a countable $\omega$-subcover. Equivalently, a space $X$ is a $\varepsilon$-space if every finite power of $X$ is
```

Comment:
the word "open" is redundant given the def of $\omega$-cover in the previous sentence.

Reply:
Addressing these now.